openehr-rm/src/main/java/com/nedap/archie/rm/support/identification/PartyRef.java

```diff
@@ -9,7 +9,7 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "PARTY_REF")
-public class PartyRef extends ObjectRef {
+public class PartyRef extends ObjectRef<ObjectId> {
```

Comment:
shouldn't this be ? extends ObjectId, since it will always be a subclass? Or does this just work?

Reply:
This does just work. PartyRef always allow any form of ObjectId. With `extends ObjectRef<? extends ObjectId>` the `setId` method would not work correctly.